generators/react/templates/src/main/webapp/app/entities/_entityFolder_/_entityFile_-update.tsx.ejs

```diff
@@ -149,12 +149,14 @@ export const <%= entityReactName %>Update = () => {
   }, [updateSuccess]);
 
   const saveEntity = values => {
-<%_ for (field of fields.filter(field => !field.id)) {
+<%_ for (field of fields) {
     const fieldName = field.fieldName;
 _%>
   <%_ if (field.fieldTypeTimed)  { _%>
     values.<%= fieldName %> = convertDateTimeToServer(values.<%= fieldName %>);
-  <%_ } _%>
+  <%_ } else if (field.fieldTypeNumeric || field.id && !field.autoGenerate) { _%>
```

Comment:
I think `field.id` should be removed from the condition.
```suggestion
  <%_ } else if (field.fieldTypeNumeric && !field.autoGenerate) { _%>
```

Reply:
ok thanks @mshima :)

Reply:
```suggestion
  <%_ } else if (field.fieldTypeNumeric) { _%>
```

autogenerated will be treated at runtime.